atlas/packages/app/src/shared/components/InfiniteVideoGrid/InfiniteVideoGrid.tsx

```diff
@@ -26,8 +26,7 @@ const InfiniteVideoGrid: React.FC<InfiniteVideoGridProps> = ({
   initialLoading,
   className,
 }) => {
-  // TODO: base this on the container width and some responsive items/row
-  const videosPerRow = VIDEOS_PER_ROW
+  const [videosPerRow, setVideosPerRow] = useState(VIDEOS_PER_ROW)
```

Comment:
Now I think it'd make sense to rename the constant to `INITIAL_VIDEOS_PER_ROW` or something alike

Reply:
Good catch, will change it